orbit/cmd/desktop/desktop.go

```diff
@@ -132,19 +134,86 @@ func main() {
 				select {
 				case <-myDeviceItem.ClickedCh:
 					if err := open.Browser(deviceURL.String()); err != nil {
-						log.Printf("open browser my device: %s", err)
+						log.Error().Err(err).Msg("open browser my device")
 					}
 				case <-transparencyItem.ClickedCh:
 					if err := open.Browser("https://fleetdm.com/transparency"); err != nil {
-						log.Printf("open browser transparency: %s", err)
+						log.Error().Err(err).Msg("open browser transparency")
 					}
 				}
 			}
 		}()
 	}
 	onExit := func() {
-		log.Println("exit")
+		log.Info().Msg("exit")
 	}
 
 	systray.Run(onReady, onExit)
 }
+
+// setupLogs configures our logging system to write logs to rolling files and
+// stderr, if for some reason we can't write a log file the logs are still
+// printed to stderr.
+func setupLogs() {
+	stderrOut := zerolog.ConsoleWriter{Out: os.Stderr, TimeFormat: time.RFC3339Nano, NoColor: true}
+
+	dir, err := logDir()
+	if err != nil {
+		log.Logger = log.Output(stderrOut)
+		log.Error().Err(err).Msg("find directory for logs")
+		return
+	}
+
+	logFile := &lumberjack.Logger{
+		Filename:   path.Join(dir, "Fleet", "fleet-desktop.log"),
```

Comment:
Don't we need to create the directory in case it doesn't exist? https://github.com/fleetdm/fleet/blob/44f902218c0c1d7d569e8a7c6884f4f8ea029c6e/orbit/cmd/orbit/orbit.go#L162-L166

Reply:
I looked into that, but `lumberjack.Logger` does that for us. I believe we're doing that explicitly in orbit because it runs as a privileged user and we need to do `secure.MkdirAll`.

https://github.com/natefinch/lumberjack/blob/47ffae23317c5951a2a6267a069cf676edf53eb6/lumberjack.go#L208-L211

Having said that, happy to do the check explicitly so we don't depend on `lumberjack`